packages/block-editor/src/store/selectors.js

```diff
@@ -2084,3 +2084,19 @@ export const __experimentalGetActiveBlockIdByBlockNames = createSelector(
 		validBlockNames,
 	]
 );
+
+/**
+ * Tells if the block with the passed clientId was just inserted.
+ *
+ * @param {Object} state Global application state.
+ * @param {Object} clientId Client Id of the block.
+ * @param {boolean} source Insertion source of the block.
```

Comment:
Agreed. I marked the source as optional and updated the type of the param 0d0164c Just to be clear since function parameters default to `undefined` it's fine to just update the documentation and leave the parameter as is?

Reply:
>Just to be clear since function parameters default to undefined it's fine to just update the documentation and leave the parameter as is?

Yeah I think so, if the default value would be different than `undefined` (here is an [example](https://github.com/WordPress/gutenberg/blob/d2e3b2b6cf8e681938403817422bbfa57c95a97d/packages/notices/src/store/selectors.js#L45-L56)) then we should set it in the param. In this case since the default is `undefined`, I think it's ok to leave it this way.